src/idstore-export/src/main.rs

```diff
@@ -1,16 +1,20 @@
 extern crate core;
 
-use base64::{engine::general_purpose, Engine as _};
-use clap::Parser;
-use merk::rocksdb;
-use merk::rocksdb::{IteratorMode, ReadOptions};
-use merk::tree::Tree;
-use std::collections::BTreeMap;
-use std::path::PathBuf;
+use {
```

Comment:
Not sure how you got this refactor... We don't use a `use { /* all imports */ }` pattern anywhere else, so if you could revert this that would be great.